### PR TITLE
Added Tests, Build Tools, Badges, Unicorns

### PR DESCRIPTION
**Pull request by dozoisch**
:dancers:

### CODE REVIEW

**Comment by dozoisch** [comment]
Only thing left for you now is activating travis build https://travis-ci.org/mathieumg/tempura and then doing the release using `npm run patch|minor|major`

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: node_js
+node_js:
```
Comment (mathieumg) [inline .travis.yml:3]: Why not 0.10?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+[![devDependency Status](https://david-dm.org/mathieumg/tempura/dev-status.svg)](https://david-dm.org/mathieumg/tempura#info=devDependencies)
+
 Template literal utilities under the form of tags
+
+# Usage
+## noMultiSpaceAfterLineFeed
+Utility to write template literals without having to worry about whitespace after newline feeds and before text, and newline feeds.
+
+```js
+import { noMultiSpaceAfterLineFeed } from 'tempura';
```
Comment (mathieumg) [inline README.md]: Would you mind not having the spaces inside the braces?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+
+let myMultilineString = noMultiSpaceAfterLineFeed`
```
Comment (mathieumg) [inline README.md]: Any reason why this is not `const`? :P

File: README.md [hunk 4]
```diff
@@ -1,2 +1,67 @@
...
+*/
+```
+
+## noWhitespace
+Utility to remove all whitespace in template literals.
+
+```js
+import { noWhitespace } from 'tempura';
+
+let myStringWithWhiteSpace = noWhitespace`
```
Comment (mathieumg) [inline README.md]: I use this one to compose long URLs, if you were curious!

File: test/tests.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import assert from "assert";
+import { EOL } from "os";
```
Comment (mathieumg) [inline test/tests.js]: Cool! Also, same remark as earlier regarding spaces in braces.

File: test/tests.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+import * as tempura from "../src";
+
+describe("Tempura", () => {
+  describe("noMultiSpaceAfterLineFeed", () => {
+    it("should remove the spaces after a line feed", () => {
+      let actual = tempura.noMultiSpaceAfterLineFeed`My Super String
```
Comment (mathieumg) [inline test/tests.js]: `const`?

File: .eslintrc [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    "browser": true,
+    "mocha": true,
+    "node": true
+  },
+  "ecmaFeatures": {
+  },
+  "parser": "babel-eslint",
+  "plugins": [
+  ],
+  "rules": {
```
Comment (mathieumg) [inline .eslintrc:13]: Maybe we should add `prefer-const` and `object-curly-spaces`? ;)

File: test/tests.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+but is multilined for line length`;
+      /* eslint-enable indent */
+      let expected = "expression that shouldbe on a single linebut is multilined for line length";
+      assert.equal(actual, expected);
+    });
+  });
+
+  describe("noWhitespace", () => {
+    it("should remove all whitespace", () => {
+      /* eslint-disable indent */
```
Comment (mathieumg) [inline test/tests.js]: It complains here too?

File: test/tests.js [hunk 9]
```diff
@@ -0,0 +1,52 @@
+import assert from "assert";
+import { EOL } from "os";
+
+import * as tempura from "../src";
```
Comment (mathieumg) [inline test/tests.js:4]: Maybe we should explicitly import the utils?

**Comment by mathieumg** [comment]
Thanks for this! :rocket: :rose: I enabled Travis, I'm not sure if it'll kick in on the next push, or only on the next PR.

File: test/tests.js (hunk 9 above)
Comment (dozoisch) [inline test/tests.js:4]: But then, for every test/function you add you change this line :p

**Comment by mathieumg** [comment]
Thanks! :tada:

**Pull request merged**